.travis.yml

```diff
@@ -25,7 +25,6 @@ notifications:
     skip_join: false
 
 before_install:
-  - sudo apt-get install libgmp3-dev
```

Comment:
Now that we seemingly aren't using `sudo`, does that mean we can potentially use the [`sudo: false`](https://github.com/mozilla/fxa-content-server/blob/f2a2d9cb5c2ecdeb70576bbab69f37d27b88b77e/.travis.yml#L12) hack to potentially speed up builds?

http://docs.travis-ci.com/user/workers/container-based-infrastructure/

/cc @vladikoff